.binder/requirements.txt

```diff
@@ -1 +1 @@
-etna[all] @ git+https://github.com/tinkoff-ai/etna.git@master
+etna[all] @ git+https://github.com/tinkoff-ai/etna.git@issue-974
```

Comment:
why do we need this?

Reply:
I will remove it, it is for the case when I want to run it on clean machine in binder (some warnings aren't present where).